instance_config.yaml

```diff
@@ -25,6 +25,14 @@ SECRET_KEY: ""
 # (set to false in prod)
 debug: false
 
+
+# JWT AUTH settings
+JWT_AUTH:
+    JWT_SECRET_KEY':
+    JWT_ISSUER':
+    JWT_AUDIENCE':
+    JWT_VERIFY_AUDIENCE':
```

Comment:
@yro  can you please confirm how we'll get the values of these variables on veda production, it should be similar to edx-platform. My main concern is like how / who will get these values from studio production and put them in video pipeline ?